```
17:07:29 /home/jenkins/android-tools//android-ndk-r20/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/i686-linux-android/4.9.x/../../../../i686-linux-android/bin/ld: error: ../tools/depends/xbmc-depends/i686-linux-android-21-debug/lib/libpython3.8.a(xmltok.o): multiple definition of '_INTERNAL_trim_to_complete_utf8_characters'
17:07:40 /home/jenkins/android-tools//android-ndk-r20/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/i686-linux-android/4.9.x/../../../../i686-linux-android/bin/ld: /home/jenkins/workspace/Android-X86-Matrix/tools/depends/xbmc-depends/i686-linux-android-21-debug/lib/libexpat.a(xmltok.o): previous definition here
```

--- a/tools/depends/target/expat/Makefile
+++ b/tools/depends/target/expat/Makefile
@@ -3,12 +3,19 @@ DEPS= ../../Makefile.include Makefile
 
 # lib name, version
 LIBNAME=expat
-VERSION=2.2.9
+VERSION=2.4.4
 SOURCE=$(LIBNAME)-$(VERSION)
-ARCHIVE=$(SOURCE).tar.bz2
+ARCHIVE=$(SOURCE).tar.xz
 
 # configuration settings
-CONFIGURE=cp -f $(CONFIG_SUB) $(CONFIG_GUESS) ./conftools; CFLAGS="-D_INTERNAL_trim_to_complete_utf8_characters=expat_INTERNAL_trim_to_complete_utf8_characters" ./configure --prefix=$(PREFIX) --disable-shared --without-examples --without-tests --without-docbook
+CONFIGURE=cp -f $(CONFIG_SUB) $(CONFIG_GUESS) ./conftools; \
+                                              CFLAGS="-D_INTERNAL_trim_to_complete_utf8_characters=expat_INTERNAL_trim_to_complete_utf8_characters" \
+                                              ./configure --prefix=$(PREFIX) \
+                                                          --disable-shared \
+                                                          --without-examples \
+                                                          --without-tests \
+                                                          --without-xmlwf \
+                                                          --without-docbook
 
 LIBDYLIB=$(PLATFORM)/lib/.libs/lib$(LIBNAME).a
 

